.github/workflows/ci-code-analysis.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         servers: '[{"id": "github", "username": "dummy", "password": "${env.GITHUB_TOKEN_REF}"}]'
         githubServer: false
     - name: Run spotbugs
-      run: mvn -B -U -Dheadless=true compile spotbugs:check
+      run: mvn -B -U -Dquarkus.quinoa=false compile spotbugs:check
```

Comment:
Actually, the project doesn't even have `spotbugs` added as a plugin in the `pom.xml` yet :-) we'll need to copy over that configuration as well.

Reply:
And just doing `mvn clean compile`, it doesn't try to build the frontend anyway, so the `quarkus.quinoa` property isn't needed.